src/SFA.DAS.EmployerCommitmentsV2/SFA.DAS.EmployerCommitmentsV2.csproj

```diff
@@ -11,9 +11,9 @@
     <PackageReference Include="SFA.DAS.Authorization.CommitmentPermissions" Version="6.0.59" />
     <PackageReference Include="SFA.DAS.Authorization.EmployerFeatures" Version="6.0.59" />
     <PackageReference Include="SFA.DAS.Authorization.EmployerUserRoles" Version="6.0.59" />
-    <PackageReference Include="SFA.DAS.CommitmentsV2.Api.Client" Version="7.27.0" />
```

Comment:
these versions are much lower than 7.29.34 in all the other project files?

Reply:
fixed